src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -77,35 +79,20 @@ public FormFullDTO process(FormRegisterDTO form, MultipartFile file) {
         return createdForm;
     }
 
-    private FormFullDTO saveForm(FormRegisterDTO formRegisterDTO) {
-
-        UUID countryId = formRegisterDTO.getCountry().getId();
-        Country country = countryRepository.findById(countryId).orElseThrow(() -> new NotFoundException("City with uuid = " + countryId +
-                " Not Found in DB", "form.uuid.invalid"));
-        UUID cityId = formRegisterDTO.getCity().getId();
-        City city = cityRepository.findById(cityId).orElseThrow(() -> new NotFoundException("City with uuid = " + cityId +
-                " Not Found in DB", "form.uuid.invalid"));
-
-        Form form = mapper.toFormEntity(formRegisterDTO);
-
-        form.setCountry(country);
-        form.setCity(city);
-
-
-        log.info("Save form, id: {} internshipId {} and status {}", form.getId(), form.getInternshipId(), FormStatus.REGISTERED);
-
-        form.setFormStatus(FormStatus.REGISTERED);
-
-        log.info("The form status is {}", FormStatus.REGISTERED);
 
-        formRepository.save(form);
+    @Override
+    public List<FormFullDTO> getAllByUserId(UUID userId) {
 
-        FormFullDTO dto = mapper.toFormDto(form);
+        MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
```

Comment:
Why do you put class name in MDC context? It should be presented in the log output by default.

Reply:
I will deal with this in the future pr, becouse on this moment it is our general approach, thanks. i created task